doc/development_process/proposals.rst

```diff
@@ -1,26 +1,133 @@
 
+Feature Tracking
+#################
+
+For feature tracking we use Github labels to classify new features and
+enhancements. The following is the description of each category:
+
+Enhancement
+  Changes to existing features that are not considered a bug and would not
+  block a release. This is an incremental enhancement to a feature that already
+  exists in Zephyr.
+
+Feature request
+  A request for a feature that is not on the roadmap yet, that has not been
+  vetted, and needs further discussion and details.
+
+Feature
+  A committed and planned feature with a detailed design and implementation
+  proposal and an owner. Features must go through an RFC process and must be
+  vetted and discussed in the TSC before a target milestone is set.
+
+The following workflow should be used to process features and roadmap items.
+
+This is the formal way for asking for a new feature in Zephyr and indicating its
+importance to the project.  Often, the requester may have a readiness and
+willingness to drive implementation of the feature in an upcoming release, and
+should assign the request to themselves.
+If not though, an owner will be assigned after evaluation by the TSC.
+A feature request can also have a companion RFC with more details on the feature
+and a proposed design or implementation.
+
+- Label new features requests as ``feature-request``
+- The TSC discusses new ``feature-request`` items regulary and triages them.
+  Items are examined for similarity with existing features, how they fit with
+  the project goals and other timeline considerations. The priority is
+  determined as follows:
+
+  - High = Next milestone
```

Comment:
This is just a question (which does not necessarily need a change, but needs a reponse :) ): Will "feature requests" be allowed priority? or only "features"?, if so, who sets the priority to a "feature request" and what would the meaning be? (the priority requester is asking the project give it?)

(if these labels only apply to "features" maybe this labels should be moved to be below the next paragraph?)

Reply:
i think it should apply to both and helps with triaging process because a request will become a release feature at some point, so setting the priority on the request keeps track of that.